### ISSUE TITLE
Make TechTV video URL's work without a slug at the end

### ISSUE THREAD

**Issue by mbertrand** [mitodl/odl-video-service#494]
The slug at the end of TechTV URL's is not mandatory, so in OVS `/videos/595` should work the same as `videos/595-chandelier'

**Comment by pdpinch** [comment]
Do we know what the URL pattern is? Will something like `/(\d+\)-\` give us id number?

**Comment by emmello** [comment]
all videos will be /videos (or /embeds) /video#   anything after that number can be used in a link or truncated and it will still play.

For instance, the latest Blossoms is:  http://techtv.mit.edu/videos/33570-los-inquilinos-que-necesita-mi-cuerpo-para-estar-sano-the-tenants-my-body-needs-to-be-healthy

A wee bit wordy to use as a link.

**Comment by mbertrand** [comment]
The pattern in `urls.py` should be`(?P<video_key>\d+)(-.+)?/?$'`

**Comment by mbertrand** [comment]
@pdpinch should I work on this yet?  Just need to make that change to `urls.py` above and then modify the unit tests a bit.

**Comment by pdpinch** [comment]
Go for it. My goal is to have everything done for the public collection migration by the end of the sprint, except for waiting for YouTube.

**Comment by mbertrand** [comment]
@emmello do you think anyone might try to use embed URL's like `/embeds/595-chandelier` instead of just `/embeds/595`?  I can tweak the url pattern for that one too, currently only `/embeds/595` will work.

**Comment by emmello** [comment]
Yes, some people copy the text, others might add their own language for a new name after the /595, so really the only important thing is just the slug up to the number.
